src/Tizen.CoreUI/src/CoreUI.Canvas/AlphaAnimation.cs

```diff
@@ -0,0 +1,264 @@
+    [CoreUI.Canvas.AlphaAnimation.NativeMethods]
+    [CoreUI.Wrapper.BindingEntity]
+    [SuppressMessage("Microsoft.Naming", "CA1724:TypeNamesShouldNotMatchNamespaces")]
+    public class AlphaAnimation : CoreUI.Canvas.Animation
+    {
+        /// <summary>Pointer to the native class description.</summary>
+        public override System.IntPtr NativeClass
+        {
+            get
+            {
+                if (((object)this).GetType() == typeof(AlphaAnimation))
+                {
+                    return GetCoreUIClassStatic();
+                }
+                else
+                {
+                    return CoreUI.Wrapper.ClassRegister.klassFromType[((object)this).GetType()];
+                }
+            }
+        }
+
+        [System.Runtime.InteropServices.DllImport(CoreUI.Libs.Evas)] internal static extern System.IntPtr
+            efl_canvas_alpha_animation_class_get();
+
+        /// <summary>Initializes a new instance of the <see cref="AlphaAnimation"/> class.
+        /// </summary>
+        /// <param name="parent">Parent instance.</param>
+        public AlphaAnimation(CoreUI.Object parent= null) : base(efl_canvas_alpha_animation_class_get(), parent)
+        {
+            FinishInstantiation();
+        }
+
+        /// <summary>Subclasses should override this constructor if they are expected to be instantiated from native code.
+        /// Do not call this constructor directly.
+        /// </summary>
+        /// <param name="ch">Tag struct storing the native handle of the object being constructed.</param>
+        protected AlphaAnimation(ConstructingHandle ch) : base(ch)
+        {
+        }
+
+        /// <summary>Initializes a new instance of the <see cref="AlphaAnimation"/> class.
+        /// Internal usage: Constructs an instance from a native pointer. This is used when interacting with C code and should not be used directly.
+        /// </summary>
+        /// <param name="wh">The native pointer to be wrapped.</param>
+        internal AlphaAnimation(CoreUI.Wrapper.WrappingHandle wh) : base(wh)
+        {
+        }
+
+        /// <summary>Initializes a new instance of the <see cref="AlphaAnimation"/> class.
+        /// Internal usage: Constructor to forward the wrapper initialization to the root class that interfaces with native code. Should not be used directly.
+        /// </summary>
+        /// <param name="baseKlass">The pointer to the base native Eo class.</param>
+        /// <param name="parent">The CoreUI.Object parent of this instance.</param>
+        protected AlphaAnimation(IntPtr baseKlass, CoreUI.Object parent) : base(baseKlass, parent)
+        {
+        }
+
+
+        /// <summary>Alpha range to animate. <c>0.0</c> means the object is transparent and <c>1.0</c> means the object is opaque.</summary>
+        /// <param name="from_alpha">Initial alpha value.</param>
+        /// <param name="to_alpha">Ending alpha value.</param>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
According to current branch policy, the master branch is related to API8 and if you want it to be released for API7 you need to change the current branch. Additionally, it may require the update of all ```<since_tizen>``` tags to the related version.